src/vs/workbench/contrib/welcome/gettingStarted/test/common/gettingStarted.test.ts

```diff
@@ -9,24 +9,14 @@ import { IStorageService } from 'vs/platform/storage/common/storage';
 
 suite('Workbench - GettingStarted', () => {
 	let instantiation: TestInstantiationService | null = null;
-	let welcomePageEnvConfig: string | null = null;
 	let hideWelcomeSettingsValue: string | null = null;
-	// let machineId: string | null = null;
-	let appName: string | null = null;
 
 	suiteSetup(() => {
 		instantiation = new TestInstantiationService();
 		instantiation.stub(IWorkspaceContextService, {
-			getConfiguration: () => {
-				return {
-					env: {
-						welcomePage: welcomePageEnvConfig,
-						appName: appName
-					}
-				};
-			}
+
 		});
-		instantiation.stub(IStorageService, {
```

Comment:
@chrmarti These tests seem empty. Can we delete the file?

Reply:
@mjbvz Yes, please. (Not my code.)